src/common.rs

```diff
@@ -188,3 +188,73 @@ pub enum UnitSectionOffset<T = usize> {
     /// An offset into the `.debug_types` section.
     DebugTypesOffset(DebugTypesOffset<T>),
 }
+
+/// An identifier for a DWARF section.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum SectionId {
+    /// The `.debug_abbrev` section.
+    DebugAbbrev,
```

Comment:
Nice, I was going to suggest adding this, but looks like you've anticipated it already :)